modules/ROOT/pages/notifications.adoc

```diff
@@ -0,0 +1,15 @@
+= Important Notifications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+== Release Announcements
```

Comment:
add something like: 

Keeping the API Community Manager components up-to-date is critical for the optimal operation of your API community. You can follow new releases in the channels below so you can plan upgrading [LINK TO UPGRADING SECTION] your installation at your convenience.

Reply:
Good point. Added.